spec/subscription_list_spec.rb

```diff
@@ -3,10 +3,12 @@
 module QueueBus
   describe SubscriptionList do
     describe ".from_redis" do
-      it "should return from attributes" do
-        mult = {"event_one" => {"class" => "MyClass", "queue_name" => "default", "key" => "event_one", "matcher" => {"bus_event_type" => "event_one"}}, 
-                "event_two" => {"class" => "MyClass", "queue_name" => "else",    "key" => "event_two", "matcher" => {"bus_event_type" => "event_two"}}}
+      let(:mult) do
+        {"event_one" => {"class" => "MyClass", "queue_name" => "default", "key" => "event_one", "matcher" => {"bus_event_type" => "event_one"}},
+         "event_two" => {"class" => "MyClass", "queue_name" => "else",    "key" => "event_two", "matcher" => {"bus_event_type" => "event_two"}}}
+      end
```

Comment:
The test setup has two different `queue_name` values. I'm assuming that was strictly for testing that the values get set correctly and not because that's a possible scenario? From what I understand, each app gets a new `SubscriptionList` from the dispatcher.